src/ast/mod.rs

```diff
@@ -1247,6 +1247,14 @@ pub enum Statement {
         /// deleted along with the dropped table
         purge: bool,
     },
+    /// DROP Function
+    DropFunction {
+        if_exists: bool,
+        /// One or more function to drop
+        func_desc: Vec<DropFunctionDesc>,
```

Comment:
What do you think about reusing the `CreateFunctionArg` structure used in CREATE FUNCTION

https://github.com/sqlparser-rs/sqlparser-rs/blob/d420001c37eeb38aac26a85dd5661755d6ef4228/src/ast/mod.rs#L1410-L1417

?

I can also see the argument for using a separate struct. If we stay with a separate struct, I suggest we name if `DropFunctionArg` to mirror the `CreateFunctionArg`

Reply:
I'll try to combine these structures later

Reply:
sorry for my late reply。I've reconsidered your suggestion, and I think the structure might be acceptable. The `DropFunctionDesc` contains the function name and function parameters. The `CreateFunctionArg` only describes function parameters. Also, `CreateFunctionArg` has been reused and renamed `OperateFunctionArg`

Reply:
> What do you think about reusing the `CreateFunctionArg` structure used in CREATE FUNCTION
> 
> https://github.com/sqlparser-rs/sqlparser-rs/blob/d420001c37eeb38aac26a85dd5661755d6ef4228/src/ast/mod.rs#L1410-L1417
> 
> ?
> 
> I can also see the argument for using a separate struct. If we stay with a separate struct, I suggest we name if `DropFunctionArg` to mirror the `CreateFunctionArg`

sorry for my late reply。I've reconsidered your suggestion, and I think the structure might be acceptable. The `DropFunctionDesc` contains the function name and function parameters. The `CreateFunctionArg` only describes function parameters. Also, `CreateFunctionArg` has been reused and renamed `OperateFunctionArg`